libraries/AP_Scheduler/PerfInfo.cpp

```diff
@@ -29,6 +30,49 @@ void AP::PerfInfo::ignore_this_loop()
     ignore_loop = true;
 }
 
+// allocate the task info array
+void AP::PerfInfo::allocate_task_info(uint8_t num_tasks)
+{
+    _task_info = new TaskInfo[num_tasks];
+    if (_task_info == nullptr) {
+        hal.console->printf("Unable to allocate scheduler TaskInfo\n");
```

Comment:
I think the MAVLINK message is better.
What do you use to check the console output?
I do not have a communication device connected to the console.

Reply:
The scheduler is initialized pretty early, I'm not sure this will work but I could try

Reply:
Fixed